components/library/libgsasl/Makefile

```diff
@@ -35,14 +36,16 @@ include $(WS_MAKE_RULES)/common.mk
 
 PATH=$(PATH.gnu)
 
-CFLAGS+= -I/usr/include/idn
+CFLAGS += -I/usr/include/idn
 
 CONFIGURE_OPTIONS.32 += --sysconfdir=/etc
 CONFIGURE_OPTIONS.32 += --libexecdir=/usr/lib
 CONFIGURE_OPTIONS.64 += --libexecdir=/usr/lib/$(MACH64)
 CONFIGURE_OPTIONS += --disable-static
+CONFIGURE_OPTIONS += --with-gssapi-impl=mit
 
 # Auto-generated dependencies
 REQUIRED_PACKAGES += library/libidn
 REQUIRED_PACKAGES += system/library
+REQUIRED_PACKAGES += system/library/security/gss
 REQUIRED_PACKAGES += system/library/security/libgcrypt
```

Comment:
When I run gmake REQUIRED_PACKAGES here the dependency on libgcrypt will be removed. Maybe gss has replaced it?